src/gerrit.py

```diff
@@ -40,7 +40,26 @@ def __call__(self, request: PreparedRequest):
         request.prepare_cookies(self.cookie_jar)
         return request
 
-class Gerrit(object):
+
+class FakeGerrit(object):
```

Comment:
nit: I'd omit the `object`, https://google.github.io/styleguide/pyguide.html#39-classes

Reply:
Ah yes, I am still stuck on Python 2